inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -770,4 +795,32 @@ public function cancel_queues() {
 	public function delete_as_tables_transient_on_tools_page() {
 		delete_transient( 'rocket_rucss_as_tables_count' );
 	}
+
+	/**
+	 * Call the filter to check if the deletion from RUCSS is activated.
+	 *
+	 * @return mixed|null
+	 */
+	protected function is_rucss_deletion_activated() {
+		return apply_filters( 'rocket_rucss_deletion_activated', true );
```

Comment:
Add docblock for the filter https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters